test/iron-ajax.html

```diff
@@ -216,13 +215,16 @@
           expect(requests).to.deep.eql(ajax.activeRequests);
         });
 
-        test('empties `activeRequests` when requests are completed', function(done) {
+        test('empties `activeRequests` when requests are completed', function() {
           server.respond();
-          timePasses(1).then(function() {
+          return requests[0].completes.then(function() {
```

Comment:
Yeah, that would be nice. The lite polyfill doesn't include a Promise.all implementation, but I wrote a little polyfill for #70 so I'll just pull that in here.